.lando.yml

```diff
@@ -53,35 +53,26 @@ services:
   #       - /dev/shm:/dev/shm
   #     command: /opt/bin/entry_point.sh
   # elasticsearch:
-  #   type: compose
-  #   meUser: elasticsearch
-  #   ssl: true
-  #   sslExpose: false
-  #   services:
-  #     image: blacktop/elasticsearch:7
-  #     command: /elastic-entrypoint.sh elasticsearch
-  #     ports:
-  #       - "9200:9200"
-  #       - "9300:9300"
-  #     volumes:
-  #       - ./.lando/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+  #   type: elasticsearch:7
+  #   portforward: 9200
+  #   mem: 1050m
   #   run_as_root:
-  #     # Install `analysis-icu` plugin after ES service boots up
-  #     - elasticsearch-plugin install analysis-icu
-  #   overrides:
-  #     environment:
-  #       JAVA_HOME: /usr/lib/jvm/default-jvm
-  #       ES_JAVA_OPTS: -Xms512m -Xmx512m
+  #     - |
```

Comment:
Why not using https://github.com/lando/lando/blob/master/examples/elasticsearch/.lando.yml#L11-L12 here? Wasn't it working?

Reply:
Because it sometimes installs the plugin when container restarts and causes errors because the plugin is already installed. This solution checks if the plugin is already installed before installing it.

Reply:
Any chance the official containers (that are used in silta too) might work?
https://github.com/wunderio/charts/blob/master/elasticsearch/values.yaml#L53-L55

Reply:
Yes, we can try to use `docker.elastic.co` images instead of Bitnami's.

Reply:
Tried, but faced problems due the fact that ES is using Centos which doesn't match with Lando. Got also similar confirmation from Lando Slack: https://devwithlando.slack.com/archives/C2XBSHX8R/p1604451941360500
<img width="765" alt="Screenshot 2020-12-18 at 03 22 46" src="https://user-images.githubusercontent.com/146800/102562727-610c9a80-40e0-11eb-9bf3-4c514c4a77da.png">

Reply:
For the new plugins setup reference: https://github.com/bitnami/bitnami-docker-elasticsearch/blob/399c57fbe82da43f107056520b7bd2a9966baff4/7/debian-9/rootfs/libelasticsearch.sh#L406-L427